dataprofiler/profilers/json_decoder.py

```diff
@@ -20,6 +21,7 @@ def get_column_profiler_class(class_name: str) -> BaseColumnProfiler:
     """
     profiles = {
         CategoricalColumn.__name__: CategoricalColumn,
+        FloatColumn.__name__: FloatColumn,
```

Comment:
Instead of init prior to output, lets just return the class.

Reply:
done